src/test/scala/firrtlTests/DriverSpec.scala

```diff
@@ -66,7 +66,7 @@ class DriverSpec extends FreeSpec with Matchers with BackendCompilationUtilities
       val firrtlOptions = optionsManager.firrtlOptions
       val inputFileName = optionsManager.getBuildFileName("fir", firrtlOptions.inputFileNameOverride)
       inputFileName should be ("./cat.fir")
-      val outputFileName = optionsManager.getBuildFileName("v", firrtlOptions.outputFileNameOverride)
+      val outputFileName = firrtlOptions.getOutputConfig(optionsManager).asInstanceOf[SingleFile].targetFile
```

Comment:
Can you add a helper method, like `firrtlOptions.getTargetFile`? And issue a nice error message if one doesn't exist?